### ISSUE TITLE
Not right working with lot of threads specified

### ISSUE THREAD

**Issue by drdeft** [brichard19/BitCrack#32]
BitCrack hacks 202M hashes per second on GTX 1080 with -p 1024 (threads)
But really it doesn't.

I tested it on same targets with -p 128 and found 5 keys.
But cracking speed with 128 threads is 89M hashes per second.

![image](https://user-images.githubusercontent.com/1372819/45106761-b582ea80-b161-11e8-9501-ba3f24e5d37d.png)

**Comment by natedawg469** [comment]
Yeah i have been noticing that you have to play around with the parameters. I made a text file where I just change the parameters and play with it.

**Comment by natedawg469** [comment]
where did you get a file with 18 million addresses?

**Comment by drdeft** [comment]
I have parced top balances

**Comment by adminnu** [comment]
At me such file from 21 million addresses, probably you excluded 0.00001 balance, but I think it is possible to exclude up to 0.01

**Comment by bill32767** [comment]
What parameters you used here?

**Comment by adminnu** [comment]
On such a large number of addresses did not try. I've studied the code, the check is in the GPU so it does not work.

**Comment by successhalf** [comment]
help me set the command line with the instructions below

BitCrack.exe [OPTIONS] [TARGETS]

Where [TARGETS] are one or more Bitcoin address

Options:

-i, --in FILE
    Read addresses from FILE, one address per line. If FILE is "-" then stdin is read.

-o, --out FILE
    Append private keys to FILE, one per line.

Bitcrack.exe ......?
